app/templates/layout.html

```diff
@@ -6,15 +6,19 @@
   <title>BadgeYay</title>
   <link rel="stylesheet" href="{{ url_for('static', filename='css/bootstrap.css') }}">
   <link rel="stylesheet" href="{{ url_for('static', filename='css/style.css') }}">
+  <link rel="stylesheet" href="{{ url_for('static', filename='css/jquery.minicolors.css') }}">
   <link rel="shortcut icon" href="{{ url_for('static', filename='favicon.ico') }}">
+
```

Comment:
Please remove extra line spce.

Reply:
@vaibhavsingh97, I have removed the extra line space.